src/selectors.jl

```diff
@@ -12,6 +12,13 @@ findindex(int::Interval, r::AbstractVector) =
 findindex(int::Interval, r::AbstractRange{T}) where {T<:Union{Number,Char}} =
     findall(in(int), r)
 
+# find interval in a vector of intervals: same as generic findindex in lookup.jl
+function findindex(int::Interval, r::AbstractVector{<:Interval})
```

Comment:
Hmm, I suppose this is a bit ambiguous as you could also want to find the first occurrence in which `int` overlaps or intersects with one of the intervals in `r`. I suppose that's what the functional interface is for though? @mcabbott do you have a preference?